ivy/functional/frontends/numpy/statistics/percentile.py

```diff
@@ -0,0 +1,44 @@
+import ivy
+import numpy
+from ivy.func_wrapper import with_unsupported_dtypes
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+    handle_numpy_dtype,
+    from_zero_dim_arrays_to_scalar,
+    handle_numpy_out,
+    
+)
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+
+def percentile(

```

Comment:
Defining `percentile`  in order_statistics.py file is enough. No need to duplicate.

Reply:
Done